frontend/src/lib/components/PropertyFilters/PropertySelect.tsx

```diff
@@ -8,10 +8,11 @@ type PropertyOption = EventProperty
 
 interface Props {
     optionGroups: Array<PropertyOptionGroup>
-    value: PropertyOption | null
+    value: Partial<PropertyOption> | null
```

Comment:
Q: Why do we need to use partial here?

Reply:
Partial because we're getting objects like `{ value: undefined; label: undefined }` instead of null, which is why `autoOpenIfEmpty && !propertyOption` would be false, preventing the autofocus.

Reply:
Not sure I understand, but this seems like a minor enough of a typing issue.